src/main/java/net/javadiscord/javabot/listener/HugListener.java

```diff
@@ -24,6 +27,33 @@ public class HugListener extends ListenerAdapter {
 	private final AutoMod autoMod;
 	private final BotConfig botConfig;
 
+	private static final Pattern FUCKER = Pattern.compile("(fuck)(ing|er|k+)?", Pattern.CASE_INSENSITIVE);
+
+	private static String processFuck(String str) {
```

Comment:
Please move these methods down so they are together with `processHug` since they belong together.

Also, please make sure that the method names of the `processFuck` and `processHug` methods distinguish what is different between them.